code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -450,6 +453,28 @@ var/global/image/plasmaman_on_fire = image("icon"='icons/mob/OnFire.dmi', "icon_
 		H.reagents.remove_reagent(chem.id, REAGENTS_METABOLISM)
 		return 1
 
+/datum/species/plasmaman/before_equip_job(datum/job/J, mob/living/carbon/human/H)
+	var/datum/outfit/plasmaman/O = new /datum/outfit/plasmaman
+	H.equipOutfit(O)
+	return 0
 
+/datum/species/plasmaman/qualifies_for_rank(rank, list/features)
+	if(rank in command_positions)
+		return 0
+	if(rank in security_positions)
+		return 0
+	if(rank == "Clown" || rank == "Mime")//No funny bussiness
+		return 0
+	if(rank == "Bartender")
```

Comment:
@Anonus: The thought was to put them in purely work-oriented positions for starters, as players/admins were concerned about the griefing that might ensue. However if there's enough interest to have them as Bartenders I don't really see that being a big problem, although I'd prefer to keep them out of Clown and Mime.